dbt/scripts/utils/helpers.py

```diff
@@ -28,14 +28,25 @@ def create_logger(
     if log_file_path is not None and os.path.exists(log_file_path):
         os.remove(log_file_path)
 
-    # Create and start the default stream logger
-    logging.basicConfig(
-        level=logging.INFO,
-        format="%(asctime)s - %(levelname)s - %(message)s",
-        datefmt="%Y-%m-%d_%H:%M:%S.%f",
-        force=True,
```

Comment:
Something I learned while trying to figure out why the log `levelname` wasn't being included in CloudWatch: This `basicConfig` only applies to the root (default) handler, not to any handlers that you add to the logger later on. I decided to refactor to use an explicit `StreamHandler` just to make this clearer.